functions/api/branch.js

```diff
@@ -367,7 +258,11 @@ function validateParams(body, type) {
                     zipcode: joi.number()
                 }),
                 isHeadOffice: joi.bool(),
-                isActive: joi.bool()
+                isActive: joi.bool(),
+                users: joi.array()
+                    .items(joi.string().allow('')),
```

Comment:
allows empty string? why?

Reply:
Users array is never populated using an update. It will be populated by your trigger.